changelog.md

```diff
@@ -10,6 +10,12 @@ class: changelog
 
 {% capture changelog %}
 
+## Version 5.1.5 December 19, 2019
+* Fixed the UI not working with hybrid devices that accept both touch and mouse events.
+* Fixed bug with pasting image URLs when paste as text is enabled.
+* Fixed the `charmap` dialog initially focusing the first tab instead of the search input.
```

Comment:
focusing the s/first tab/first tab of the dialog/ instead of the search s/input/input field/   ?

Reply:
Fixed